### DISCUSSION TITLE
服务端分多个文件可以吗

### DISCUSSION THREAD

**Discussion by sylvoslee** [abersheeran/rpc.py#15] [Q&A]
不同文件对应一个sub route

**Comment by abersheeran** [comment]
没有对文件做限制，你怎么组织代码都行。

**Comment by sylvoslee** [comment]
看子路由的创建方式是RPC(prefix="/sub-route/")，那多个子路由，是不是就有多个RPC实例，那怎样用Uvicorn启动多个rpc实例？

**Reply by abersheeran** [reply]
用这个把多个实例组合到一个 application：https://baize.aber.sh/wsgi#subpaths

**Comment by sylvoslee** [comment]
谢谢
